pages/parentenrollclass.tsx

```diff
@@ -0,0 +1,142 @@
+                <input {...input} />
+                <Box
+                    {...checkbox}
+                    lineHeight="24px"
+                    fontSize="16px"
+                    fontWeight="normal"
+                    textColor="#0C53A0"
+                    width="340px"
+                    height="50px"
+                    border="2px solid #E1E1E1"
+                    borderRadius="6px"
+                    _checked={{
+                        fontWeight: "normal",
+                        bg: "white",
+                        textColor: "#0C53A0",
+                        border: "2px solid #0C53A0",
+                    }}
+                    textAlign="center"
+                    py="13px"
+                    my="10px"
+                >
+                    {props.children}
+                </Box>
+            </Box>
+        );
+    }
+    function ChildrenButtons() {
+        // children to be retrieved from parent enroll endpoint
+        const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+
+        const { getRootProps, getRadioProps } = useRadioGroup({
+            name: "framework",
+            defaultValue: "react",
+            onChange: () => setIsDisabled(false),
+        });
+
+        const group = getRootProps();
+
+        return (
+            <Stack {...group}>
+                {children.map((value) => {
+                    const radio = getRadioProps({ value });
+                    return (
+                        <RadioCard key={value} {...radio}>
+                            {value}
+                        </RadioCard>
+                    );
+                })}
+            </Stack>
+        );
+    }
+
+    return (
+        <Wrapper>
+            <Link href="/" ml="100%">
+                <CloseIcon />
+            </Link>
+
+            <Center>
+                <Text mt={"15px"} fontWeight={"700"} fontSize={"36px"}>
+                    Program Registration
+                </Text>
+            </Center>
+            <Center>
+                <Text pb="55px" align="center" mt="30px">
+                    Who would you like to register for{" "}
+                    <b>
+                        Building Bridges with Music - <br />
+                        Singing Monkeys (Ages 9 and under)?{" "}
+                    </b>
+                    (select one)
+                </Text>
+            </Center>
+
+            <Center>
+                <ChildrenButtons />
+            </Center>
+            <Center mt="45px" mb="200px">
+                <Button
+                    height="50px"
+                    width="340px"
+                    borderRadius="6px"
+                    background="#737373"
+                    fontWeight="normal"
+                    textColor="#FFFFFF"
+                    fontSize="16px"
+                    isDisabled={isDisabled}
+                >
+                    {/* Link to the next page in the flow (does not exist yet) */}
```

Comment:
Will the other steps of parent enrollment be in this file as well?

Reply:
No, I talked to Mahad/Stacy about this when the ticket was assigned and he said it would just be this page

Reply:
Right, but I'm asking if it will be in this file as well, not if it's in the same task

Reply:
I am really sorry, I misunderstood, I think the other pages will be in this file as well